### ISSUE TITLE
midi control 

### ISSUE THREAD

**Issue by na7681** [vargaradu/UA-Midi-Control#2]
<img width="547" alt="Captura de pantalla 2019-06-12 a la(s) 11 12 35" src="https://user-images.githubusercontent.com/34495873/59358650-50b57b80-8d03-11e9-8952-3209b8632d7f.png">

Hi vargaradu. Thanks for your app. I´m having some trouble in controling the main volume in my Focusrite Scarlett 18i20 2nd gen. The mapping with my midi device works ok and I set the permissions in F control but when moving the faders in mi BCF2000 nothing happens. I also have the f control closed as you said. Thanks for your help !!!

**Comment by raduvarga** [comment]
Hi! Did you click "Stop Midi Map" and then try your faders?

**Comment by na7681** [comment]
Yes yes, I tried that. But still no response. Thank you very much !

**Comment by na7681** [comment]
I´m on High sierra by the way.

**Comment by raduvarga** [comment]
@na7681 I've just released a completely new version for Focusrite that's much more stable and has more features, if you're curious you can check it out here: https://www.raduvarga.com/software/focusrite-midi-control
